### PR TITLE
Adding a development section

### PR DESCRIPTION
**Pull request by kwridan**
- The development section contains information on how to build and test tuist

### CODE REVIEW

**Comment by kwridan** [comment]
Have I missed anything? Happy to also iterate on the language / style of the guide.

File: contribution/development.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
+description: >-
+  Getting setup for developing tuist
+---
+
+# Setup
+
+tuist is a Swift package which requires [Swift Package Manager](https://swift.org/package-manager/) (Swift PM) to build and develop it. If you already have [Xcode](https://developer.apple.com/xcode/) isntalled, then you are set! See the [Swift PM installation section](https://github.com/apple/swift-package-manager#installation) for details on alternte methods of setting it up.
```
Comment (pepibumur) [inline contribution/development.md]: Nitpick: `tuist` ~> `Tuist`

Comment (pepibumur) [reply contribution/development.md]: Typo: `isntalled`

File: contribution/development.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+```sh
+gem install bundler
+bundle install
+```
+
+Once setup, you can run the acceptance tests via:
+
+```sh
+rake features
```
Comment (pepibumur) [inline contribution/development.md]: `bundle exec rake features`, otherwise it'll use the Rake from the system.

**Comment by kwridan** [comment]
Done! 

yikes should have used an markdown editor with a spell checker :p

**Pull request merged**